MIP/MIP-60/README.md

```diff
@@ -0,0 +1,234 @@
+# MIP-60: Crosschain bridge architectures
+- **Description**: This MIP provides some background on crosschain bridges designs and trade-offs related to security, decentralization, and performance.
+- **Authors**: [Franck Cassez](mailto:franck.cassez@movementlabs.xyz), [Andreas Penzkofer](mailto:andreas.penzkofer@movementlabs.xyz)
+
+## Abstract
+
+Crosschain bridges allow assets to be transferred between different blockchains. This MIP provides some background on crosschain bridge designs and the trade-offs related to security, decentralization, and performance.
+
+## Motivation
+
+To transfer assets from Ethereum to Move Rollup (and back), we have to employ a _bridge_. Ideally we would re-use an existing bridge, but there is no available bridge template from Ethereum to Move-based Rollups, so we have to design one.
+Designing a bridge between two blockchains is a complex task. As pointed out in [Bridges – Ethereum Foundation](https://ethereum.org/en/developers/docs/bridges/#integrating-bridges):
+
+> 1. **Building your own bridge** – Building a secure and reliable bridge is not easy, especially if you take a more trust-minimized route. Moreover, it requires years of experience and technical expertise related to scalability and interoperability studies. Additionally, it would require a hands-on team to maintain a bridge and attract sufficient liquidity to make it feasible.
+
+To design our bridge, it may be useful to understand the main types of architectures, their strengths and weaknesses. It can also help us reflect on the different discussions and proposals (MIPs, MDs) and how they relate to the different architectures.
+
+
+## Definitions and Bridge introduction
+
+Bridge terminology is frequently ambiguous as the following shows but we attempt to clarify the terms used in this MIP.
+
+**Base classes of bridges** [[5]][REF5]
```

Comment:
IMO it would be good to have the dimensions/attributes along which these vary listed in a table s.t. we can see how we arrive at this particular number of classes.